pymc_marketing/mmm/budget_optimizer.py

```diff
@@ -50,8 +50,8 @@ class BudgetOptimizer(BaseModel):
         The adstock class.
     saturation : SaturationTransformation
         The saturation class.
-    num_days : int
-        The number of days.
+    num_periods : int
+        The number of time units.
```

Comment:
In general, I think this doc string (not just this line) could use with a bit more elaboration. For example, the number of time units...what? But given that this is largely a bug fix, I say merge it and we can improve docstrings later.